ts/sagas/contentLoaders.ts

```diff
@@ -29,7 +29,16 @@ function getServiceMetadata(
   return new Promise((resolve, _) =>
     contentClient
       .getService({ serviceId })
-      .then(resolve, () => resolve(undefined))
+      .then(resolve, () =>
+        resolve(
+          left([
```

Comment:
Should you create **_left errors_** as done in _ts/sagas/backendInfo.ts_?
```javascript
const error: Errors = [
    {
      context: [],
      value: "some error occurred"
    }
  ];
```

Reply:
Yes in more clear using a constant. Fin in 2d98acf0